content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -96,7 +96,7 @@ rules:
   - <list of policies to authorize>
 ```
 
-Then the `(Cluster)Role` is bound to the authorized user(s):
```

Comment:
The original intention was `Role` or `ClusterRole`, right?

Reply:
That's how I'd read it.

Reply:
I found the original text confusing if i can write it as `Role or ClusterRole`

Reply:
> I found the original text confusing if i can write it as `Role or ClusterRole`

Hi @sftim @tengqm  any thought on this.

Reply:
Not sure why it is confusing. It is clear to me as it is.

Reply:
in [this](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#via-rbac) we are using ClusterRole as a example or before example we write like **Role or ClusterRole**  so for uniformity  of the docs i suggested the changes, so that it would easy for the new user to understand. if you think no need to change it then i am OK with that.

Reply:
(Cluster)Role _not_ in backticks might be easier. I think the issue is mainly for localizers and for non-native English speakers; if English is your main language then you probably recognize the idiom from general reading.